pkg/server/sessions/manager.go

```diff
@@ -195,37 +196,29 @@ func (sm *manager) expireSessions() {
 
 	now := time.Now()
 
-	idleSessCount := 0
+	inactiveSessCount := 0
 	sm.logger.Debugf("checking at %s", now.Format(time.UnixDate))
 	for ID, sess := range sm.sessions {
-		if sess.GetLastHeartBeat().Add(sm.options.MaxSessionIdleTime).Before(now) && sess.GetStatus() != Idle {
-			sess.setStatus(Idle)
-			sm.logger.Debugf("session %s became Idle, no more heartbeat received", ID)
-		}
-		if sess.GetLastActivityTime().Add(sm.options.MaxSessionIdleTime).Before(now) && sess.GetStatus() != Idle {
-			sess.setStatus(Idle)
-			sm.logger.Debugf("session %s became Idle due to max inactivity time", ID)
+		if sess.GetLastActivityTime().Add(sm.options.MaxSessionInactivityTime).Before(now) && sess.GetStatus() != inactive {
+			sess.setStatus(inactive)
+			sm.logger.Debugf("session %s became Inactive due to max inactivity time", ID)
 		}
 		if sess.GetCreationTime().Add(sm.options.MaxSessionAgeTime).Before(now) {
-			sess.setStatus(Dead)
-			sm.logger.Debugf("session %s exceeded MaxSessionAgeTime and became Dead", ID)
+			sess.setStatus(dead)
+			sm.logger.Debugf("session %s exceeded MaxSessionAgeTime and became dead", ID)
 		}
-		if sess.GetStatus() == Idle {
-			idleSessCount++
+		if sess.GetStatus() == inactive {
+			inactiveSessCount++
```

Comment:
Minor: inactiveSessCount should only be incremented if the session is not set to dead in the `if` below